src/tools/cleaner.js

```diff
@@ -110,9 +110,39 @@ const createBundleBank = (appRaw, event = {}) => {
 
 const maskOutput = output => _.pick(output, 'results', 'status');
 
+const normalizeEmptyRequestField = (shouldCleanup, field) => req => {
```

Comment:
Kept this as a callback to an array iterator function like `forEach`. We _could_ pull that part in here and have a simple call to normalize that loops for the supplied field from request.

Reply:
I think I like the idea of centralizing the call to `Object.entries` so we don't need to keep re-typing that. Doesn't look like there's any place where the calling function _needs_ to change that callback directly.